### PR TITLE
Task/bws37 private routes

### PR DESCRIPTION
**Pull request by sofia0602**

### CODE REVIEW

**Review by Yaroslaww-1 — DISMISSED** [review]

File: frontend/src/components/auth/auth.tsx [hunk 1]
```diff
@@ -5,31 +5,25 @@ import {
...
-import { useLocation, useAppDispatch, useAppSelector } from 'hooks/hooks';
+import { useLocation, useAppDispatch } from 'hooks/hooks';
 import { SignInForm, SignUpForm } from './components/components';
 import styles from './auth.module.scss';
 import logo from 'assets/img/logo.svg';
-import { Navigate } from 'components/common/common';
+import { useNavigate } from 'react-router-dom';
 
 const Auth: FC = () => {
-  const { user } = useAppSelector(({ auth }) => ({
```
Comment (what1s1ove) [inline frontend/src/components/auth/auth.tsx]: there was a right logic. move it back

File: frontend/src/components/common/private-route/private-route.tsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { FC } from 'react';
 import { RouteProps } from 'react-router-dom';
 import { AppRoute } from 'common/enums/enums';
 import { useAppSelector } from 'hooks/hooks';
-import { Navigate } from 'components/common/common';
+import { Navigate } from 'react-router-dom';
```
Comment (what1s1ove) [inline frontend/src/components/common/private-route/private-route.tsx]: there was a right import

File: frontend/src/components/common/common.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 export { Routes, Route } from 'react-router-dom';
 export { ErrorMessage } from '@hookform/error-message';
-export { Navigate } from 'react-router-dom';
+export { Navigate, useNavigate } from 'react-router-dom';
```
Comment (what1s1ove) [inline frontend/src/components/common/common.ts:3]:
```suggestion
export { Navigate } from 'react-router-dom';
```

File: frontend/src/store/auth/reducer.ts [hunk 4]
```diff
@@ -24,10 +24,12 @@ const reducer = createReducer(initialState, (builder) => {
   builder.addCase(signUp.rejected, (state) => {
     state.dataStatus = DataStatus.REJECTED;
   });
+
   builder.addCase(signIn.fulfilled, (state, action) => {
     state.dataStatus = DataStatus.FULFILLED;
     state.user = action.payload;
   });
+
```
Comment (what1s1ove) [inline frontend/src/store/auth/reducer.ts:32]:
```suggestion
  builder.addCase(signIn.fulfilled, (state, action) => {
    state.dataStatus = DataStatus.FULFILLED;
    state.user = action.payload;
  });
```

File: frontend/src/components/auth/auth.tsx [hunk 5]
```diff
@@ -20,7 +20,6 @@ const Auth: FC = () => {
   if (hasUser) {
     return <Navigate to={AppRoute.ROOT} />;
   }
-
```
Comment (what1s1ove) [inline frontend/src/components/auth/auth.tsx:23]: add a new line

**Pull request closed without merging**